syntax = "proto3";



message NotifyRequest{
    string message = 1;
    string subject = 2;
    repeated string recipients = 3;
    string message_type = 5;
    repeated string notification_channels = 6;
}

message NotifyResponse{
    string message = 1;
    string subject = 2;
    repeated string recipients = 3;
    string status = 5;
    string error = 6;
    repeated string notification_channels = 7;

}

service NotifierService{
    rpc Notify(NotifyRequest) returns (NotifyResponse){}
}
